installer/systemConfigurationHelper.php

```diff
@@ -0,0 +1,363 @@
+        array('cou_code' => 'TR','name' => 'TURKEY','cou_name' => 'Turkey','iso3' => 'TUR','numcode' => '792'),
+        array('cou_code' => 'TM','name' => 'TURKMENISTAN','cou_name' => 'Turkmenistan','iso3' => 'TKM','numcode' => '795'),
+        array('cou_code' => 'TC','name' => 'TURKS AND CAICOS ISLANDS','cou_name' => 'Turks and Caicos Islands','iso3' => 'TCA','numcode' => '796'),
+        array('cou_code' => 'TV','name' => 'TUVALU','cou_name' => 'Tuvalu','iso3' => 'TUV','numcode' => '798'),
+        array('cou_code' => 'UG','name' => 'UGANDA','cou_name' => 'Uganda','iso3' => 'UGA','numcode' => '800'),
+        array('cou_code' => 'UA','name' => 'UKRAINE','cou_name' => 'Ukraine','iso3' => 'UKR','numcode' => '804'),
+        array('cou_code' => 'AE','name' => 'UNITED ARAB EMIRATES','cou_name' => 'United Arab Emirates','iso3' => 'ARE','numcode' => '784'),
+        array('cou_code' => 'GB','name' => 'UNITED KINGDOM','cou_name' => 'United Kingdom','iso3' => 'GBR','numcode' => '826'),
+        array('cou_code' => 'US','name' => 'UNITED STATES','cou_name' => 'United States','iso3' => 'USA','numcode' => '840'),
+        array('cou_code' => 'UM','name' => 'UNITED STATES MINOR OUTLYING ISLANDS','cou_name' => 'United States Minor Outlying Islands','iso3' => NULL,'numcode' => NULL),
+        array('cou_code' => 'UY','name' => 'URUGUAY','cou_name' => 'Uruguay','iso3' => 'URY','numcode' => '858'),
+        array('cou_code' => 'UZ','name' => 'UZBEKISTAN','cou_name' => 'Uzbekistan','iso3' => 'UZB','numcode' => '860'),
+        array('cou_code' => 'VU','name' => 'VANUATU','cou_name' => 'Vanuatu','iso3' => 'VUT','numcode' => '548'),
+        array('cou_code' => 'VE','name' => 'VENEZUELA','cou_name' => 'Venezuela','iso3' => 'VEN','numcode' => '862'),
+        array('cou_code' => 'VN','name' => 'VIET NAM','cou_name' => 'Viet Nam','iso3' => 'VNM','numcode' => '704'),
+        array('cou_code' => 'VG','name' => 'VIRGIN ISLANDS, BRITISH','cou_name' => 'Virgin Islands, British','iso3' => 'VGB','numcode' => '92'),
+        array('cou_code' => 'VI','name' => 'VIRGIN ISLANDS, U.S.','cou_name' => 'Virgin Islands, U.s.','iso3' => 'VIR','numcode' => '850'),
+        array('cou_code' => 'WF','name' => 'WALLIS AND FUTUNA','cou_name' => 'Wallis and Futuna','iso3' => 'WLF','numcode' => '876'),
+        array('cou_code' => 'EH','name' => 'WESTERN SAHARA','cou_name' => 'Western Sahara','iso3' => 'ESH','numcode' => '732'),
+        array('cou_code' => 'YE','name' => 'YEMEN','cou_name' => 'Yemen','iso3' => 'YEM','numcode' => '887'),
+        array('cou_code' => 'ZM','name' => 'ZAMBIA','cou_name' => 'Zambia','iso3' => 'ZMB','numcode' => '894'),
+        array('cou_code' => 'ZW','name' => 'ZIMBABWE','cou_name' => 'Zimbabwe','iso3' => 'ZWE','numcode' => '716')
+    );
+
+    /**
+     * Returns Country List
+     * @return array
+     */
+    function getCountryList() {
+        $list = array();
+        $list[''] = '-Select-';
```

Comment:
￼Is this part of the country list ?  i think this should be part of the country widget